integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/util/AbstractIT.java

```diff
@@ -239,6 +241,20 @@ public static void startIntegrationTest() throws Exception {
     } else {
       client = GravitinoAdminClient.builder(serverUri).build();
     }
+
+    Awaitility.await()
+        .atMost(60, TimeUnit.SECONDS)
+        .pollInterval(5, TimeUnit.SECONDS)
```

Comment:
You haven't changed it, 5 seconds seems too long.

Reply:
Oh, sorry. I will change it.